doc/luigi_patterns.rst

```diff
@@ -183,19 +183,17 @@ resource:
 
 .. code-block:: python
 
-    class DailyOverwriteTask(luigi.Task):
+    class A(luigi.Task):
         date = luigi.DateParameter(batch_mode=max)
 
         resources = {'overwrite_resource': 1}
 
-Now if you have multiple tasks such as
-DailyOverwriteTask(date=2016-06-01),
-DailyOverwriteTask(date=2016-06-02),
-DailyOverwriteTask(date=2016-06-03), the scheduler will just tell you to
-run the highest available one and mark the lower ones as batch_running.
-Using a unique resource will prevent multiple tasks from writing to the
-same location at the same time if a new one becomes available while
-others are running.
+Now if you have multiple tasks such as ``A(date=2016-06-01)``,
+``A(date=2016-06-02)``, ``A(date=2016-06-03)``, the scheduler will just
+tell you to run the highest available one and mark the lower ones as
+batch_running. Using a unique resource will prevent multiple tasks from
+writing to the same location at the same time if a new one becomes
+available while others are running.
```

Comment:
While it's implicit from this text if you think hard about it. We perhaps should mention that "if task X, Y, Z gets batch-run as task Z, those 3 tasks will only take up as many resources as Z alone would have had". (and perhaps the word resources is a hyperlink to the docs about luigi resources)